corehq/apps/style/static/dashboard/less/dashboard.less

```diff
@@ -52,6 +52,7 @@
   .panel-heading {
     font-weight: bold;
     font-size: @font-size-large;
+    text-align: center;
```

Comment:
Did this both because the title looks odd with the extra-wide button and coincidentally to fix bug with popovers being cut off:
![screen shot 2015-05-06 at 1 47 34 pm](https://cloud.githubusercontent.com/assets/1486591/7500003/03298070-f3f9-11e4-88ca-96e54107a8b7.png)